### ISSUE TITLE
User reported issue: parallel arrangements of batteries causes improper current sharing

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/circuit-construction-kit-common#677]
> **Problem description**
> I was expecting to be able to break the simulation, but this particular issue likely will be one of concern to your target audience:
>  
> I was playing with parallel arrangements of batteries in the “Circuit Construction Kit: DC - Virtual Lab” sim an found the results were prone to diode-like behavior and improper current sharing under default values.
> The image attached shows an extreme failure of the simulator, however, it seems most common structures with batteries in parallel will lead to issues. I suspect this is a consequence of the sigmoid function used to limit output current with respect to set voltage, rather than a true (linear) resistance. One can go to great lengths to devise a matched cable resistance system and the simulation is still prone to misbehave.
>  
> **Steps to reproduce**
> Put two batteries in parallel and short them. Alternatively use a resistive load. See Picture for extreme case.
![I_Broke_it](https://user-images.githubusercontent.com/5090275/109540416-a275da80-7a7f-11eb-8dc8-4664a7a38b91.PNG)

**Comment by arouinfar** [comment]
@samreid can you take a look at this? My guess is that introducing internal resistance to the batteries when shorted/high current is likely a factor here.

**Comment by samreid** [comment]
Could this be explained by the top circuit having resistance from the wires, and the middle branch having no resistance contribution from wires?

**Comment by samreid** [comment]
On hold while we investigate https://github.com/phetsims/circuit-construction-kit-common/issues/284

**Comment by samreid** [comment]
This will be addressed in https://github.com/phetsims/circuit-construction-kit-common/issues/676, I recommend closing this issue, but we will need a reminder to notify the client.  I'll add something to https://github.com/phetsims/circuit-construction-kit-common/issues/676
